docker-compose.yml

```diff
@@ -37,6 +37,18 @@ services:
       RUNESTONE_HOST: ${RUNESTONE_HOST}
       CERTBOT_EMAIL: ${CERTBOT_EMAIL}
       LOAD_BALANCER_HOST: '${LOAD_BALANCER_HOST}'
+      # Stripe settings.
+      STRIPE_PUBLISHABLE_KEY: ${STRIPE_PUBLISHABLE_KEY}
+      STRIPE_SECRET_KEY: ${STRIPE_SECRET_KEY}
+      STRIPE_DEV_PUBLISHABLE_KEY: ${STRIPE_DEV_PUBLISHABLE_KEY}
+      STRIPE_DEV_SECRET_KEY: ${STRIPE_DEV_SECRET_KEY}
+      STRIPE_TEST_PUBLISHABLE_KEY: ${STRIPE_TEST_PUBLISHABLE_KEY}
+      STRIPE_TEST_SECRET_KEY: ${STRIPE_TEST_SECRET_KEY}
+      # E-mail settings.
+      EMAIL_SERVER: ${EMAIL_SERVER}
```

Comment:
Are these email env vars necessary?  If we define them we ought to use them in 1.py maybe?  I know it makes them available for the new bookserver as well, but we are not doing anything with email there yet.

Reply:
I'm trying to put all secrets in one place -- `.env`. The e-mail config includes a password, so I put all e-mail stuff together there.

I will update `1.py.prototype` to make use of these -- thanks! I somehow overlooked that.

Reply:
I just pushed fixes.

Reply:
This now contains the fix for the LTI bug. (I forgot an import -- sigh.)

Reply:
How in the world could this have been working for me all week without this!!??!!

Reply:
:)

Reply:
My earlier LTI "fix" really broke everything, sigh. But since the exception is caught and logged, it's not as obvious.

Reply:
Because we love how web2py handles imports! :-(